x-pack/plugins/security_solution/public/common/components/event_details/alert_summary_view.tsx

```diff
@@ -68,20 +79,34 @@ const fields = [
   { id: 'signal.threshold_result.cardinality', label: ALERTS_HEADERS_THRESHOLD_CARDINALITY },
 ];
 
-const processFields = [
-  ...fields,
+const processCategoryFields: EventSummaryField[] = [
+  ...defaultDisplayFields,
   { id: 'process.name' },
   { id: 'process.parent.name' },
   { id: 'process.args' },
 ];
 
-const networkFields = [
-  ...fields,
+const networkCategoryFields: EventSummaryField[] = [
+  ...defaultDisplayFields,
   { id: 'destination.address' },
   { id: 'destination.port' },
   { id: 'process.name' },
 ];
 
+const memoryShellCodeAlertFields: EventSummaryField[] = [
```

Comment:
suggestion: This one looks like an extension of `memorySignatureAlertFields` on line 105. Perhaps reuse it like so:

```
const memoryShellCodeAlertFields: EventSummaryField[] = [ 
   ... memorySignatureAlertFields, 
   {
      id: 'Target.process.thread.Ext.start_address_details.memory_pe.imphash',
      label: ALERTS_HEADERS_TARGET_IMPORT_HASH,
    }
]
```